app/gui/view/debug_scene/cloud-dashboard/src/lib.rs

```diff
@@ -37,131 +37,483 @@
+                    let name = &project.name;
+                    let state = view::project::TaggedState::from(state.clone());
+                    let state = match state {
+                        view::project::TaggedState::New => "New".to_string(),
+                        view::project::TaggedState::Created => "Created".to_string(),
+                        view::project::TaggedState::OpenInProgress => "OpenInProgress".to_string(),
+                        view::project::TaggedState::Opened => "Opened".to_string(),
+                        view::project::TaggedState::Closed => "Closed".to_string(),
+                    };
+                    format!("({state}) {name}")
+                }
+                Columns::LastModified => LAST_MODIFIED.to_string(),
+                // FIXME [NP]: Display icons for users/groups with access to the project and their
+                // corresponding permissions (e.g., read/write/execute).
+                Columns::SharedWith => SHARED_WITH.to_string(),
+                // FIXME [NP]: Display icons for the project's labels. Labels may be user-defined
+                // or system-defined (e.g., labels indicating high resource usage or outdated
+                // version).
+                Columns::Labels => LABELS.to_string(),
+                // FIXME [NP]: Display icons for datasets associated with the project, as well as
+                // what permissions are set on the dataset, from the user's perspective.
+                Columns::DataAccess => DATA_ACCESS.to_string(),
+                // FIXME [NP]: Display which usage plan the project is configured for (e.g.,
+                // "Interactive" or cron-style, etc.).
+                Columns::UsagePlan => USAGE_PLAN.to_string(),
+            };
+
+            let entry_model = EntryModel {
+                text:           model.into(),
+                disabled:       Immutable(false),
+                override_width: Immutable(if col == 1 && row == 5 { Some(10.0) } else { None }),
+            };
+            Some((row, col, entry_model))
+        }
 
-            root.add_child(&text);
-            let base_shape = rounded_box::View::new();
-            root.add_child(&base_shape);
+        fn set_projects(&self, projects: Vec<view::project::Project>) {
+            *self.projects.borrow_mut() = projects;
 
-            Self { application, root, base_shape, text }
+            let row = self.rows();
+            let col = Columns::LEN;
+            self.grid.resize_grid(row, col);
         }
 
-        fn set_size(&self, size: Vector2) {
-            self.base_shape.size.set(size);
+        fn refit_grid_to_shape(&self, shape: &ensogl::display::scene::Shape) {
+            // FIXME [NP]: Make the margins into StyleWatchFrp variables?
+            const TOP_MARGIN: f32 = 10f32;
+            const BOTTOM_MARGIN: f32 = 10f32;
+            const LEFT_MARGIN: f32 = 10f32;
+            const RIGHT_MARGIN: f32 = 10f32;
```

Comment:
move to the style

Reply:
Will be done in future PR. Opened pivotal issue to track.